src/benchmark/tpch_benchmark.cpp

```diff
@@ -102,16 +109,51 @@ int main(int argc, char* argv[]) {
     }
   }
 
-  std::cout << "- TPCH scale factor is " << scale_factor << std::endl;
+  std::cout << "- " << (jcch ? "JCC-H" : "TPC-H") << " scale factor is " << scale_factor << std::endl;
   std::cout << "- Using prepared statements: " << (use_prepared_statements ? "yes" : "no") << std::endl;
 
   // Add TPCH-specific information
   context.emplace("scale_factor", scale_factor);
   context.emplace("use_prepared_statements", use_prepared_statements);
 
-  auto item_runner = std::make_unique<TPCHBenchmarkItemRunner>(config, use_prepared_statements, scale_factor, item_ids);
-  auto benchmark_runner = std::make_shared<BenchmarkRunner>(
-      *config, std::move(item_runner), std::make_unique<TPCHTableGenerator>(scale_factor, config), context);
+  auto table_generator = std::unique_ptr<AbstractTableGenerator>{};
+  auto item_runner = std::unique_ptr<AbstractBenchmarkItemRunner>{};
+
+  if (jcch) {
+    // Different from the TPC-H benchmark, where the table and query generators are immediately embedded in Hyrise, the
+    // JCC-H implementation calls those generators externally. This is because we would get linking conflicts if we were
+    // to include both generators. Unfortunately, this approach is somewhat slower (30s to start SF1 with TPC-H, 1m18s
+    // with JCC-H).
+
+    // Try to find dbgen/qgen binaries
+    auto jcch_dbgen_path =
+        std::filesystem::canonical(std::string{argv[0]}).remove_filename() / "third_party/jcch-dbgen";
+    Assert(std::filesystem::exists(jcch_dbgen_path / "dbgen"),
+           std::string{"JCC-H dbgen not found at "} + jcch_dbgen_path.c_str());
+    Assert(std::filesystem::exists(jcch_dbgen_path / "qgen"),
+           std::string{"JCC-H qgen not found at "} + jcch_dbgen_path.c_str());
+
+    // Create the jcch_data directory (if needed) and generate the jcch_data/sf-10.0 path
```

Comment:
jcch_data/sf-**0.01** path ?

Reply:
Well, depends on which parameter you use ;)
Changed to `sf-...`.